stressTesting/artillery.yml

```diff
@@ -0,0 +1,21 @@
+config:
+  target: "http://localhost:8080"
+  phases:
+    - duration: 20
+      arrivalCount: 20
+    # - duration: 120
+    #   arrivalRate: 30
+    #   rampTo: 50
+    # - duration: 60
+    #   arrivalRate: 30
+  payload:
+    path: "./locationids.csv"
```

Comment:
You may want to consider having one list of random location IDs, and then a smaller list of 'popular' locationIDs -- listings that you anticipate showing to users often.